src/sessions.ts

```diff
@@ -650,93 +696,102 @@ export class ClientSession
     options?: TransactionOptions
```

Comment:
There's a documentation requirement for the convenient transactions API: 

```typescript
Drivers MUST document that the remaining timeout will not be applied to callback operations that do not use the ClientSession.
Drivers MUST also document that overriding timeoutMS for operations executed using the explicit session inside the provided callback will result in a client-side error, as defined in Validation and Overrides.
```

Reply:
Good catch, added